dolphinscheduler-data-quality/src/main/java/org/apache/dolphinscheduler/data/quality/DataQualityApplication.java

```diff
@@ -64,9 +65,16 @@ public static void main(String[] args) throws Exception {
             config.put(SPARK_APP_NAME, dataQualityConfiguration.getName());
         }
 
-        SparkRuntimeEnvironment sparkRuntimeEnvironment = new SparkRuntimeEnvironment(config);
+        boolean hiveClientSupport = dataQualityConfiguration
+                .getReaderConfigs()
+                .stream()
+                .anyMatch(line -> line.getType().equalsIgnoreCase(HIVE.name()));
```

Comment:
oh, I got you point, you mean when user reader do not have hive reader, we can ignore using hive support in spark context, it is not business for run success with or without hive client

Reply:
Yes, because I communicate with some users, some users rely on data quality features, but their data warehouse is not dependent on ApacheHive, so it is not installed.